easybuild/easyblocks/c/cuda.py

```diff
@@ -310,4 +333,9 @@ def make_module_req_guess(self):
             'CPATH': inc_path,
         })
 
+        if os.path.exists(os.path.join(self.installdir, 'pkgconfig')):
```

Comment:
No need to make this conditional: EasyBuild will first check whether the `pkgconfig` subdirectory exists because actually updating `$PKG_CONFIG_PATH` in the generated module file already

Reply:
fixed in 0202a4f